app/uploaders/attachment_uploader.rb

```diff
@@ -0,0 +1,19 @@
+class AttachmentUploader < CarrierWave::Uploader::Base
+  storage :file
+
+  process :save_content_type_and_size_in_model
+
+  def save_content_type_and_size_in_model
```

Comment:
This model name should be changed. I forgot to update it when I added the file name to the Upload model.